src/components/header.js

```diff
@@ -37,6 +38,20 @@ const renderLevels = ({ onClickHandler, pageData, activeLevel }: Props) => {
     }
   }
 
+  if (5 > pageData.levels) {
+    const newWidth = 20 * pageData.levels
+    const newMargin = 100 - newWidth
+
+    const SpreadDiv = styled.div`
+      display: inline-flex;
+      width: ${newWidth}%;
+      margin-left: ${newMargin}%;
+      justify-content: space-between;
+    `
```

Comment:
we-do-it-this-way-at-monzo type nit: 
we'd standardly define our styled components at the top level and pass in `newWidth` and `newMargin` as props. this makes them a little easier to reuse and semantically a tad easier to reason about as components. 

as that we could this sort of thing below
```
<SpreadDiv newWidth={newWidth} newMargin={newMargin}>{toRender}</SpreadDiv>
```

you can read a little more here for the props pattern for styled components if ya like 🙂 
https://www.styled-components.com/docs/basics#adapting-based-on-props

Reply:
I would normally agree - this is the structure I've followed throughout at `components/styles.js` - it's all styled components based on props. However, this div will never ever be used anywhere else, and the values are specific to the fact that the normal spacing on the div should be justified around ~5 levels (hence the values of 100 and 20, to make sure it looks consistent and not float to the left). I think putting it in a separate file without context adds unnecessary complexity for little benefit in this situation, but I'm not _too_ committed to this approach so if you feel strongly enough I'll change it :)

Reply:
I was suggesting the top of this file rather than a seperate one 👌 

I do see what you mean, but I can see this tripping up monzo web engineers a little as it's not something we do. so for the sake of readability and idiomaticness, probably best at the top level 👍

Reply:
Totally understand 👍
Not sure if what i've done is what you meant, but I think it improves readability :)